rudder-jetty/SOURCES/rudder-jetty.conf

```diff
@@ -25,7 +25,7 @@ then
 fi
 
 # Check JVM major version
-JAVA_MAJOR_VERSION=`${JAVA} -version 2>&1 | grep -E "(java|openjdk) version" | sed 's%.*version \"\([0-9]\.[0-9]\)\.[0-9].*\"%\1%'`
+JAVA_MAJOR_VERSION=`${JAVA} -version 2>&1 | grep -E "(java|openjdk) version" | sed 's%.*version "\([0-9]\+\.[0-9]\+\|[0-9]\+-\).*%\1%'`
```

Comment:
This regex will keep the '-'

Reply:
Removing it should to the trick since only the first match will be kept